crypto/cloudkms/signer.go

```diff
@@ -0,0 +1,63 @@
+package cloudkms
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/onflow/flow-go-sdk/crypto"
+	"github.com/onflow/flow-go-sdk/crypto/cloudkms"
+)
+
+var _ crypto.Signer = &KeyRotationSigner{}
+
+type KeyRotationSigner struct {
+	mux        sync.RWMutex
+	kmsSigners []*cloudkms.Signer
+	index      int
+	signersLen int
+}
+
+func NewSignerForKeys(
+	ctx context.Context,
+	client *cloudkms.Client,
+	keys []cloudkms.Key,
+) (*KeyRotationSigner, error) {
+	if len(keys) == 0 {
+		return nil, fmt.Errorf("no asymmetric signing keys provided")
+	}
+
+	kmsSigners := make([]*cloudkms.Signer, len(keys))
+	for i, key := range keys {
+		kmsSigner, err := client.SignerForKey(ctx, key)
+		if err != nil {
+			return nil, fmt.Errorf(
+				"could not create signer for key with ID: %s",
+				key.KeyID,
```

Comment:
also add the error in the end: `"could not create signer for key with ID: %s: %w"`

Reply:
Good point :+1: 
Added in https://github.com/onflow/flow-evm-gateway/pull/282/commits/f9303971f9c0a580a52904b3cd8e33edef785b53.
Did the same for the bootstrapping code as well.